site-root/cbl/powertools/reset-baselines.php

```diff
@@ -0,0 +1,173 @@
+<?php
+
+use Slate\CBL\Competency;
+use Slate\CBL\ContentArea;
+use Slate\CBL\StudentCompetency;
+
+use Slate\People\Student;
+
+
+$GLOBALS['Session']->requireAccountLevel('Developer');
```

Comment:
I usually open powertools to Admins+

```suggestion
$GLOBALS['Session']->requireAccountLevel('Administrator');
```